lib/lrama/options.rb

```diff
@@ -1,15 +1,27 @@
+# rbs_inline: enabled
 # frozen_string_literal: true
 
 module Lrama
   # Command line options.
   class Options
-    attr_accessor :skeleton, :header, :header_file,
-                  :report_file, :outfile,
-                  :error_recovery, :grammar_file,
-                  :trace_opts, :report_opts,
-                  :diagnostic, :y, :debug, :define,
-                  :diagram, :diagram_file, :profile_opts
+    attr_accessor :skeleton #: String
+    attr_accessor :header #: bool
+    attr_accessor :header_file #: String
+    attr_accessor :report_file #: String
+    attr_accessor :outfile #: String
+    attr_accessor :error_recovery #: bool
+    attr_accessor :grammar_file #: String
```

Comment:
It's initialized with `nil` then `String?`.

Reply:
Fixed: https://github.com/ruby/lrama/pull/527/commits/7e064dea26300767aa3d30e7fb49a72985c8a55b